nixos/modules/programs/regreet.nix

```diff
@@ -58,9 +58,100 @@ in
         modifiable properties.
       '';
     };
+
+    theme = {
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.gnome-themes-extra;
+        defaultText = lib.literalExpression "pkgs.gnome-themes-extra";
+        description = ''
+          The package that provides the theme given in the name option.
+        '';
+      };
+
+      name = lib.mkOption {
+        type = lib.types.str;
+        default = "Adwaita";
+        description = ''
+          Name of the theme to use for regreet.
+        '';
+      };
+    };
+
+    iconTheme = {
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.adwaita-icon-theme;
+        defaultText = lib.literalExpression "pkgs.adwaita-icon-theme";
+        description = ''
+          The package that provides the icon theme given in the name option.
+        '';
+      };
+
+      name = lib.mkOption {
+        type = lib.types.str;
+        default = "Adwaita";
+        description = ''
+          Name of the icon theme to use for regreet.
+        '';
+      };
+    };
+
+    font = {
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.cantarell-fonts;
+        defaultText = lib.literalExpression "pkgs.cantarell-fonts";
+        description = ''
+          The package that provides the font given in the name option.
+        '';
+      };
+
+      name = lib.mkOption {
+        type = lib.types.str;
+        default = "Cantarell 16";
```

Comment:
Perhaps this should be 2 options, `name` and `size`?

Reply:
Fixed :+1: